packages/testlab/test/testlab.acceptance.js

```diff
@@ -0,0 +1,8 @@
+const should = require('should');
+const testlab = require('..');
+
+describe('testlab', () => {
```

Comment:
Well it's using the real compiled files, but yes a true test would have to require the dep from NPMJS (ie. add testlab as a true dep via `npm install -S testlab`).